src/visualizer/visualizer.js

```diff
@@ -16,11 +16,25 @@ const week = "week";
 const month = "month";
 const year = "year";
 
+const getMaxPriorityElement = prioritized => {
+    const max_priority = [prioritized[0]]
+    prioritized.slice(1).forEach((item) => {
+        if (item.priority === max_priority[0].priority) {
+            max_priority.push(item)
+        }
+    })
+
+    var index = Math.floor((Math.random() - 0.001) * max_priority.length)
```

Comment:
Can you elaborate more on what's is `0.001` ? Or probably put it on a variable with a good name leads to better clarity

Reply:
Used as a threshold. Have added comments for more information.